### DISCUSSION TITLE
分布式键值存储 Dynamo 的实现原理 · /dynamo

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#508] [Announcements]
https://draveness.me/dynamo

**Comment by byronz** [comment]
楼主的digram做得不错，是用什么做的？

**Comment by mugbya** [comment]
回答楼上： Sketch，博主在  BigTable 那篇文章说过。

博主把 Dynamo 跟 BigTable都讲了个遍，真心厉害

**Comment by dawncold** [comment]
{% include related/distributed-system.md %}

**Comment by Tsumida** [comment]
百度了一下dynamo，发现[搜狐上一篇文章](https://www.sohu.com/a/298142999_315839)与博主你的文章极度相似...

**Comment by draveness** [comment]
> 百度了一下dynamo，发现[搜狐上一篇文章](https://www.sohu.com/a/298142999_315839)与博主你的文章极度相似...

这个就是复制粘贴的吧
